src/routes/basket.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import checkToken from  'express-jwt';
+import { addBasket } from '@entities/Basket/Basket.controller';
+
+const router = Router();
+
+
+router
+    // .get('', getBaskets)
+    // .get('/:id', getBasketById)
+    .post('', checkToken({ secret: `${process.env.JWT_PRIVATE_KEY}`, algorithms: ['HS256']}), addBasket)
```

Comment:
Only logged in user is able to create a basket, it can be moved to MeRouter, the same as you do for getBasketById and I will do for getBaskets (by name)